daemon/api_debug.go

```diff
@@ -131,7 +137,19 @@ func collectChangeTimings(st *state.State, changeID string) (map[string]*changeT
 
 	m := map[string]*changeTimings{}
 	for _, t := range chg.Tasks() {
+		lane := 0
+		// determine lowest lane number, we are only interested in a single lane here
+		for _, l := range t.Lanes() {
+			if l < lane {
```

Comment:
this needs testing, afaict  lane starts = 0 so this would happen only if l is negative

Reply:
Thanks, this was indeed wrong. Fixed and added a test.